core/src/main/java/org/infinispan/commands/ReplicableCommand.java

```diff
@@ -54,6 +54,7 @@ default Object invoke() throws Throwable {
     *
     * @return the method id of this command.  This is compatible with pre-2.2.0 MethodCall ids.
     */
+   // TODO do we just ignore this? Not possible to use ProtoStreamTypeIDs as it's a byte
    byte getCommandId();
```

Comment:
yes, I would deprecate it and add a default implementation returning zero. 
We need to fix our code where we rely on this, but in a follow up PR.

Reply:
Issue created for a follow up PR: https://github.com/infinispan/infinispan/issues/14114